build/prepare-versioning.ps1

```diff
@@ -0,0 +1,20 @@
+Write-Host "=========================="
+Write-Host "Applying versioning to GitVersion.yml..."
+
+$filename = "GitVersion.yml"
+$newContent = Get-Content $filename
+
+if ($env:BUILD_SOURCEBRANCHNAME -eq "master" -and $env:ReleaseVersioning -eq "true")
+{
+	Write-Host "Removing preview tag"
+	# Release builds do not need the preview tag
+	$newContent = $newContent -replace "tag: preview", ""
```

Comment:
we still need preview tag for release builds because MSAL is not GA.

Reply:
Good point. We might need it later, though, so I'll just comment it out.